### PR TITLE
chore: use new amountMath

### PR DESCRIPTION
**Pull request by katelynsills**

### CODE REVIEW

**Review by erights — APPROVED** [review]
LGTM

File: contract/src/otcDesk.js [hunk 1]
```diff
@@ -11,7 +10,7 @@ import {
 import { E } from '@agoric/eventual-send';
 
 /**
-
+
```
Comment (Chris-Hibbert) [inline contract/src/otcDesk.js:13]: oops?  looks like the blank line should just go away.  (I know, totally a nit.)

**Review by Chris-Hibbert — APPROVED** [review]
Do the yarn.lock changes make sense to you?  Babel was dropped in a few places in package.json, but '@babel` was added to yarn.lock. There are other additions and duplications that mean nothing to me, and don't seem related to the clean-ups in the package.json files.

**Comment by katelynsills** [comment]
> Do the yarn.lock changes make sense to you? Babel was dropped in a few places in package.json, but '@babel` was added to yarn.lock. There are other additions and duplications that mean nothing to me, and don't seem related to the clean-ups in the package.json files.

Yup, you can see what dependences are required for the different packages installed. These are related to the eslint changes.

**Pull request merged**